Library/Formula/libffi.rb

```diff
@@ -34,8 +34,8 @@ def install
     # Move lib64/* to lib/ on Linuxbrew
     lib64 = Pathname.new "#{lib}64"
     if lib64.directory?
-      system "mv #{lib64}/* #{lib}/"
-      rmdir lib64
+      mv Dir.glob('#{lib64}/*'), '#{lib}'
+      rmdir '#{lib64}'
```

Comment:
Without this change `brew audit libffi` complains about a call to system's `mv` command